src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,72 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
+
+    public int calculate(String inputString){
+        for(String splitData : inputString.split(" ")){
+            if(isSign(splitData)){
+                doArithmeticOperation(splitData);
+            } else {
+                numbers.push(Integer.parseInt(splitData));
+            }
+        }
+        return numbers.pop();
+    }
+
+    private void doArithmeticOperation(String splitData) {
```

Comment:
think for some other name than `splitData`. again it tells nothing about the purpose of var